optaplanner-quarkus-integration/pom.xml

```diff
@@ -24,43 +24,10 @@
   <url>https://www.optaplanner.org</url>
 
   <properties>
-    <version.io.quarkus>1.6.0.Final</version.io.quarkus>
-
-    <!-- TODO remove if https://issues.redhat.com/browse/PLANNER-2005 is fixed for 8.x -->
-    <!-- TODO Extending kie-parent 7 is extremely dangerous and messy because it's not aligned to quarkus -->
-    <version.asm>7.3.1</version.asm>
-    <version.commons-cli>1.4</version.commons-cli>
-    <version.commons-codec>1.13</version.commons-codec>
-    <version.com.google.guava>27.0.1-jre</version.com.google.guava>
-    <version.com.google.inject.guice>4.2.1</version.com.google.inject.guice>
-    <version.org.apache.httpcomponents.httpclient>4.5.12</version.org.apache.httpcomponents.httpclient>
-    <version.org.apache.httpcomponents.httpcore>4.4.13</version.org.apache.httpcomponents.httpcore>
-    <version.org.apache.maven>3.6.3</version.org.apache.maven>
-    <version.org.apache.maven.plugin-tools>3.6.0</version.org.apache.maven.plugin-tools>
-    <version.org.apache.maven.wagon>3.3.3</version.org.apache.maven.wagon>
-    <version.org.codehaus.plexus.plexus-classworlds>2.6.0</version.org.codehaus.plexus.plexus-classworlds>
-    <version.org.codehaus.plexus.plexus-containers>2.1.0</version.org.codehaus.plexus.plexus-containers>
-    <version.org.codehaus.plexus.plexus-interpolation>1.25</version.org.codehaus.plexus.plexus-interpolation>
-    <version.org.codehaus.plexus.plexus-utils>3.2.1</version.org.codehaus.plexus.plexus-utils>
-    <version.org.eclipse.sisu>0.3.4</version.org.eclipse.sisu>
-    <version.org.jsoup>1.11.3</version.org.jsoup>
-    <version.org.junit>5.6.2</version.org.junit>
-    <version.org.sonatype.plexus.plexus-sec-dispatcher>1.4</version.org.sonatype.plexus.plexus-sec-dispatcher>
-    <version.org.wildfly.common>1.5.4.Final-format-001</version.org.wildfly.common>
-    <version.org.ow2.asm>8.0.1</version.org.ow2.asm>
-    <version.org.jboss.jandex>2.1.3.Final</version.org.jboss.jandex>
-
-
-    <version.org.jboss.resteasy>4.5.3.Final</version.org.jboss.resteasy>
-    <version.com.fasterxml.jackson>2.10.3</version.com.fasterxml.jackson>
-    <version.com.fasterxml.jackson.databind>2.10.3</version.com.fasterxml.jackson.databind>
+    <version.io.quarkus>1.7.0.Final</version.io.quarkus>
```

Comment:
The quarkus version should be defined in optaplanner-build-parent I think, but we can't import the quarkus-bom there, not even in depMgmt, because it's a platform bom (not a user bom), so it contains groupId's from other artifacts, messing up our dep tree.